cmd/operator-sdk/up/local.go

```diff
@@ -183,7 +183,7 @@ func buildLocal(outputBinName string) error {
 	}
 	opts := projutil.GoCmdOptions{
 		BinName:     outputBinName,
-		PackagePath: filepath.Join(scaffold.ManagerDir, scaffold.CmdFile),
```

Comment:
Because the `scaffold.ManagerDir` is just the relative path, this won't work ([see constants](https://github.com/operator-framework/operator-sdk/blob/260113a850b072089c03f7193419d32ac22c491e/internal/pkg/scaffold/constants.go#L26-L27)). To make this work, you would need to include the absolute path to the package.

What made you propose the changes in the first place, were you running into any problems?

Reply:
Hi @LiliC, thank you for the quick response and detailed explanation. Really appreciated.

When I run `operator-sdk up local` to do some local test, some `function not defined` errors occurred. It's weird because `go build cmd/manager` passed. I checked the source code operator and found out that the `operator-sdk up local` only build single go file `cmd/manager/main.go` instead of the whole cmd/manager package. I believe it would be better if we build the whole package since there might be some other *.go files inside `cmd/manager`. In my case, some helper functions.

Reply:
What we could do, is similiar to what we do when `operator-sdk build`command is called -> https://github.com/operator-framework/operator-sdk/blob/51d28df214440d97031368d7d700a2805161015d/cmd/operator-sdk/build/cmd.go#L182-L190

Reply:
Mainly referring to this line:
```go
 	PackagePath: filepath.Join(projutil.CheckAndGetProjectGoPkg(), scaffold.ManagerDir), 

```

Reply:
@LiliC Thank you so much! That's everything I need for this PR!

Reply:
@chechiachang And I think this should have solved your problems as well?

Reply:
@LiliC Yes, totally! Thank you for walking me through this PR:)